.github/check-image-exists.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+checkImageExists() {
+    USER=$1
+    REPOSITORY=$2
+    IMAGE_TAG=$3
+
+    imageTagPageHttpResponse=`curl --silent -f -lSL https://index.docker.io/v1/repositories/${USER}/${REPOSITORY}/tags/${IMAGE_TAG} > /dev/null ; echo $?`
```

Comment:
It took me a while when I figure out it is working based on exit code.

Would not it make more sence to do a HEAD request and get a http response code?

```
HTTP_RESPONSE_CODE=$(curl -I https://index.docker.io/v1/repositories/${USER}/${REPOSITORY}/tags/${IMAGE_TAG} | head -n 1 | cut -d$' ' -f2)
```